Resources/Prototypes/_NF/Shipyard/decadedove.yml

```diff
@@ -0,0 +1,30 @@
+- type: vessel
+  id: DecadeDove
+  name: NT Decade Dove
```

Comment:
I think the name is too on-the-nose. Maybe just the 'Dove' or similar, or some unrelated name. The silhouette and design still clearly indicates the reference, we dont need to be that heavy handed.